spec/requests/subscriptions_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe "Jobseeker saves jobs", type: :request do
+  let(:school) { create(:school) }
+  let(:vacancy) { create(:vacancy) }
+  let(:origin) { "/jobs/#{vacancy.id}/#{vacancy.slug}" }
+
+  before do
+    vacancy.organisation_vacancies.create(organisation: school)
```

Comment:
Do you prefer this way to setting it in the factory (e.g `organisation_vacancies_attributes: ...`)

Reply:
organisation_vacancies_attributes is more minimalist, let's use it!